examples/jenkins/jenkins-persistent-template.json

```diff
@@ -204,11 +195,35 @@
       "value": "password"
     },
     {
+      "name": "MEMORY_REQUEST",
+      "displayName": "Memory Lower Limit",
+      "description": "Reserved amount of memory for the container.",
+      "value": "1200Mi"
+    },
+    {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory Upper Limit",
+      "description": "Maximum amount of memory the container can use.",
+      "value": "2000Mi"
+    },
```

Comment:
this isn't an acceptable value for online, it would consume the user's entire memory quota.  why do you think the value needs to be this high by default?